### PR TITLE
Changed isNoteworthy to Boolean

### PR DESCRIPTION
**Pull request by josedemouramachado**
To convert isNoteworthy to Boolean I used:

`isNoteworthy = (isNoteworthy == 'true')`

### CODE REVIEW

**Comment by swarajpure** [comment]
Hi @josedemouramachado 
Thank you for your pull request in Real Dev Squad!

In open source, it's always encouraged that we comment on the issue that we wish to work on *before* we start working on it. Otherwise let's say 4 people see the issue, they don't comment on the issue and they start working independently, no person knows that someone else is also working on it. In that case we'd end up with 4 pull requests and only 1 can be merged. It wastes the effort of other 3 people, right?
Therefore we should always get issues assigned to ourselves OR at least comment that `Hey, I'd like to work on this issue.` or something similar. I hope that makes sense 🙂 

Since the person whom the issue was assigned to, did not start working *before* you sent the PR, we will consider your PR 🥳 

---------------------

We (Real Dev Squad) are a community of developers consisting on college students and working professionals. We communicate over Discord. You might want to join the community, you'll get A LOT to learn 😄
We have got projects like this repository. There are projects which use simple HTML, CSS and vanilla JavaScript and also projects based on React, Ember and Vue for frontend and on Node for backend. Since we're open source, you can show your work which you do here to the interviewers as well 🎉 
No matter what your expertise level is, we've always got something for everyone! ⚡ 

Invitation link: https://discord.gg/dycQderh

**Comment by josedemouramachado** [comment]
Hello @swarajpure,

Thank you so much for the feedback!

I had no idea we should announce our interest but it makes complete sense! 😄

Glad to have my PR considered! Thank you!

File: script.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ taskForm.onsubmit = async (e) => {
       dinero: lossRateDinero,
       neelam: lossRateNeelam,
     },
-    isNoteworthy: isNoteworthy || false,
+    isNoteworthy: isNoteworthy = (isNoteworthy == 'true'),
```
Comment (aman7870) [inline script.js:65]: Hi @josedemouramachado , **isNoteworthy** returns "on" if it's checked and "undefined" if it's not checked. Please do changes according to it.

**Review by aman7870 — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (josedemouramachado) [reply script.js]: Hello @aman7870,

My bad. Changed to isNotworthy = (isNoteworthy == 'on')

**Review by whyDontI — APPROVED** [review]
Thanks for the contributions @josedemouramachado

**Pull request merged**